Lib/fontbakery/profiles/fontwerk.py

```diff
@@ -31,22 +36,39 @@ def leave_this_one_out(checkid):
         'com.google.fonts/check/vendor_id',
         'com.google.fonts/check/fstype',
         'com.google.fonts/check/gasp',
+        'com.google.fonts/check/font_copyright',
+        'com.google.fonts/check/name/line_breaks',  # can be ignored because of general fontwerk name table test.
+        'com.google.fonts/check/varfont/has_HVAR',  # not needed for fontwerk fonts.
+        'com.google.fonts/check/glyf_nested_components',  # it seem to be not an issue anymore. Maybe downgrade to 'WARN'. Tested it in various browsers, InDesign, exported a PDF, printed it.
+        'com.google.fonts/check/transformed_components',  # it seem to be not an issue anymore. Maybe downgrade to 'WARN'.
 
         # The following check they may need some improvements
         # before we decide to include it:
         'com.google.fonts/check/family/italics_have_roman_counterparts',
+        'com.google.fonts/check/varfont_instance_names',  # need improvements for optical size name or own test.
+        'com.google.fonts/check/smart_dropout',  # improvements needed when it comes to variable fonts. No hinting for var fonts should be ok.
+        'com.google.fonts/check/STAT/gf-axisregistry',  # need improvements.
+        '''
```

Comment:
Is this supposed to be a comment? I think it ends as a string next to the check-ids, so it's at least a bit "dirty", even if it doesn't break something. I would suggest using `#` for each line